### PR TITLE
DE191 Using ng-if to show email field with or without autofocus

### PR DESCRIPTION
**Pull request by shankx50**
I admit it is somewhat of a creative fix, but this was the only solution that worked for me.

### CODE REVIEW

File: crossroads.net/app/register/register_form.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <form id="nav_login" name="registerForm" role="form" ng-submit="register()" novalidate>
 
         <div class="soft-half-bottom" ng-class="{'has-error': firstnameError() }">
-              <input id="{{passwordPrefix}}-firstname" class="form-control" id="firstname" name="firstname" type="text" ng-model="newuser.firstname" required placeholder="First Name" autofocus/>
```
Comment (IngageBrewerM) [inline crossroads.net/app/register/register_form.html]: Why was the closing / removed from this element?

**Comment by IngageBrewerM** [comment]
Did you try this solution in multiple browsers?

File: crossroads.net/app/register/register_form.html (hunk 1 above)
Comment (shankx50) [inline crossroads.net/app/register/register_form.html]: I did test it in Chrome, Safari and Firefox

The closing '/' is not required for an HTML5 input tag, but it is for XHTML, so it's really a matter of personal preference.

**Comment by IngageBrewerM** [comment]
:+1:

**Pull request merged**